addons/cargo/functions/fnc_getDelayItem.sqf

```diff
@@ -0,0 +1,24 @@
+#include "..\script_component.hpp"
+/*
+ * Author: tcvm
+ * Gets the delay duration an item should take to load/unload.
+ *
+ * Arguments:
+ * 0: Item <OBJECT>
+ *
+ * Return Value:
+ * Item load/unload duration <NUMBER>
+ *
+ * Example:
+ * cursorObject call ace_cargo_fnc_getDelayItem
+ *
+ * Public: Yes
+ */
+
+params ["_item"];
+
+if (_item getVariable [QGVAR(delay), -1] >= 0) exitWith {
```

Comment:
```suggestion
if ((_item isEqualType objNull) && {_item getVariable [QGVAR(delay), -1] >= 0}) exitWith {
```

Reply:
cargo can just be a classname, 
e.g. spare wheels aren't objects until they are unloaded